activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -111,15 +111,25 @@ def build_redis_client(**redis_options)
 
       # Creates a new Redis cache store.
       #
-      # There are four ways to provide the Redis client used by the cache: the
-      # +:redis+ param can be a Redis instance or a block that returns a Redis
-      # instance, or the +:url+ param can be a string or an array of strings
-      # which will be used to create a Redis instance or a +Redis::Distributed+
-      # instance.
+      # There are a few ways to provide the Redis client used by the cache:
+      #
+      # 1. The +:redis+ param can be:
+      #    - A Redis instance.
+      #    - A +ConnectionPool+ instance wrapping a Redis instance.
+      #    - A block that returns a Redis instance.
+      #
+      # 2. The +:url+ param can be:
+      #    - A string used to create a Redis instance.
+      #    - An array of strings used to create a +Redis::Distributed+ instance.
+      #
+      # If the final Redis instance is not already a +ConnectionPool+, it will
+      # be wrapped in one using +ActiveSupport::Cache::Store::DEFAULT_POOL_OPTIONS+.
```

Comment:
I really tried to get RDoc to link to this constant (without `+`) but didn't have much luck...

Reply:
Yeah that is really weird, I also tried to no avail, maybe a bug in RDoc. But interesting that `RedisCacheStore` inherits from `Store` so maybe that is why it is confused.